### PR TITLE
Popup dialogs update

### PR DESCRIPTION
**Pull request by LanskovNV**
Added fonts and themes support

### CODE REVIEW

File: src/containers/MapPage/MapPage.js [hunk 1]
```diff
@@ -76,36 +77,51 @@ class MapsScreen extends Component {
...
     const locale = this.props.locale.toLocaleUpperCase();
+    const theme = this.props.theme;
+
     this.setState({
       showDialog : true,
       dialog: (
       <PopupDialog
-        dialogTitle={<DialogTitle title = {museum.name[locale]}/>}
+        dialogStyle={{backgroundColor: colors.BASE[theme],
+          height: 220,
```
Comment (ponjoru) [inline src/containers/MapPage/MapPage.js]: define the magic variable: 220

File: src/containers/MapPage/MapPage.js [hunk 2]
```diff
@@ -76,36 +77,51 @@ class MapsScreen extends Component {
...
           />
         ]}
       >
         <View>
-          <Text>
+          <Text style = {{color: colors.TEXT[theme], textAlign: 'center', marginTop: 20}}>
             <Image
-              align={"top"}
-              style={{width: 200, height: 200}}
+              style={{width: 230, height: 230}}
```
Comment (ponjoru) [inline src/containers/MapPage/MapPage.js]: Define the magic variables: 230

File: src/containers/MapPage/MapPage.js (hunk 1 above)
Comment (LanskovNV) [reply src/containers/MapPage/MapPage.js]: fixed

File: src/containers/MapPage/MapPage.js (hunk 2 above)
Comment (LanskovNV) [reply src/containers/MapPage/MapPage.js]: I've changed it so now it correctly works in ios and android

**Pull request merged**